miden-lib/asm/sat/miden_wallet.masm

```diff
@@ -0,0 +1,56 @@
+use.miden::sat::account
+use.miden::sat::tx
+
+#! Add a single asset to an account's vault 
+#!
+#! Stack: [ASSET]
+#! Output: []
+#!
+#! - ASSET is the non-fungible asset of interest.
+export.receive_asset
+    exec.account::add_asset
+    dropw
+end
+
+#! Creates a note which sends a single asset to the specified recipient.
+#!
+#! Stack: [ASSET, tag, RECIPIENT]
+#! Output: []
+#!
+#! - ASSET is the non-fungible asset of interest.
+#! - tag is the tag to be included in the note.
+#! - RECIPIENT is the recipient of the note.
+export.send_asset
```

Comment:
Similar comment to above. In this case, if `create_note` modifies the stack we could just indicate that the top 9 slots contain "garbage" on return.

Reply:
I now return the `ptr` -> pointer to memory where the note is stored. I assume the wallet can access this memory location (defined in the account). Is this correct?